src/Console/Command/CreateRequestsCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Console\Command;
+
+use GuzzleHttp\Client as GuzzleClient;
+use Http\Adapter\Guzzle6\Client as GuzzleAdapter;
+use OpenEuropa\EPoetry\EPoetryClientFactory;
+use OpenEuropa\EPoetry\Serializer\RequestsSerializer;
+use OpenEuropa\EPoetry\Type\CreateRequests;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CreateRequestsCommand extends Command
+{
+    protected static $defaultName = 'create-requests';
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Build and send a CreateRequests to ePoetry.')
+            ->setHelp('')
+
+            ->addOption('endpoint', null, InputOption::VALUE_REQUIRED, 'URL of the WSDL endpoint.')
+            ->addOption('in-format', null, InputOption::VALUE_OPTIONAL, 'Format for the input.', 'xml')
+            ->addOption('out-format', null, InputOption::VALUE_OPTIONAL, 'Format for the output.', 'xml')
+
+            ->addArgument('request-file', InputArgument::REQUIRED, 'Path to a file containing the body of the request.');
+    }
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $guzzle = new GuzzleClient();
```

Comment:
I would automatically discover the http client if any.

Reply:
Guzzle is listed among the dependencies so it's ok to use it here.

Reply:
:+1:  I will create a ticket to do this improvement in the future.